### ISSUE TITLE
Wrong Plugin in the dependencies-list of the plugin.xml

### ISSUE THREAD

**Issue by ir-tech** [katzer/cordova-plugin-local-notifications#514]
The device-plugin dependency is wrong in the plugin.xml

Because of this, the plugin causes the build on iOS to fail.

See in this commit they changed the package name of the device plugin: https://github.com/apache/cordova-plugin-device/commit/b58b5edd401359845b6b0539c0d0cccb996e5719

This should be taken into the plugin.xml

**Comment by SMJ93** [comment]
Hi ir-tech,

I encountered this problem too.

if you cd into your project directory run `cordova save plugins --experimental` in the command line it will try and update the XML using the installed plugins. 

This error will be output: `"Skipping org.apache.cordova.device, not a top-level plugin"` 

They need to update the example project with a new plugin / XML structure.

To get it working I created my own Cordova project using the [command line interface](https://cordova.apache.org/docs/en/4.0.0/guide_cli_index.md.html) and installed the three plugins toast, device and local notifications manually. I then copied the HTML, CSS, JavaScript files across from the kitchen sink example and it worked for me.

Best of luck!

Regards, Stu
